multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt

```diff
@@ -32,6 +32,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Davey Scans", "https://daveyscans.com", "id", overrideVersionCode = 1),
         SingleLang("Mangasusu", "https://mangasusu.co.in", "id", isNsfw = true, overrideVersionCode = 1),
         SingleLang("Manhua Raw", "https://manhuaraw.com", "en"),
+        SingleLang("ManhwaIndo", "https://manhwaindo.id", "id", isNsfw = true, overrideVersionCode = 1),
```

Comment:
```suggestion
        SingleLang("ManhwaIndo", "https://manhwaindo.id", "id", isNsfw = true),
```

New extensions in generators always start with `overrideVersionCode = 0`, which is the default value if not provided.